src/build123d/build_common.py

```diff
@@ -241,10 +243,9 @@ def edges(self, select: Select = Select.ALL) -> ShapeList[Edge]:
             ShapeList[Edge]: Edges extracted
         """
         if select == Select.ALL:
-            edge_list = self._obj.edges()
+            return self._obj.edges()
```

Comment:
IIRC pylint doesn't like multiple returns, that's why there's an intermediate variable.